layouts/index.html

```
@@ -1,26 +1,27 @@
-    
-    {{ range $notes }}
+    {{ range $posts }}
     <li>
         <span>
-            <i>
-                <time datetime='{{ .Date.Format "2006-01-02" }}' aria-label="Date: {{ .Date.Format (default "2006-01-02" .Site.Params.dateFormat) }}">
-                    {{ .Date.Format (default "2006-01-02"
-                    .Site.Params.dateFormat) }}
-                </time>
-            </i>
+            <time datetime='{{ .Date.Format "2006-01-02" }}' aria-label="Date: {{ .Date.Format (default "2006-01-02" .Site.Params.dateFormat) }}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)